packages/core/src/balance/Balance.utils.ts

```diff
@@ -8,12 +8,42 @@ import { formatBalance } from '@polkadot/util'
 
 export const KILT_COIN = new BN(1)
 
-export function formatKiltBalance(amount: BN): string {
-  return formatBalance(amount, {
+// Exported options from polkadot/util
+export interface Options {
+  decimals?: number
+  forceUnit?: string
+  withSi?: boolean
+  withSiFull?: boolean
+  withUnit?: boolean | string
+}
+
+export function formatKiltBalance(
+  amount: BN,
+  additionalOptions?: Options
+): string {
+  const options = {
     decimals: 15,
     withSiFull: true,
     withUnit: 'KILT',
-  })
+    ...additionalOptions,
+  }
+  return formatBalance(amount, options)
+}
+
+export function formatKiltBalanceDecimalPlacement(
+  amount: BN,
+  decimal: number,
+  denomination: number
+): string {
+  const decimalBN = new BN(decimal)
+  const denominationBN = new BN(denomination)
+
+  const balanceFactoring = new BN(10).pow(denominationBN.sub(decimalBN))
+  const powersOfTen = new BN(10).pow(decimalBN).toNumber()
+  const amountFactoring = amount.div(balanceFactoring).toNumber()
+
+  const amountDenomination = amountFactoring / powersOfTen
+  return amountDenomination.toFixed(decimal)
```

Comment:
As I understood it we want to be able to return the input as string with set decimal point and number of decimal places
Maybe shorten it a bit and remove the unnecessary const declarations:

```suggestion
  const factoring = new BN(10).pow(new BN(denomination - decimal))
  const skimmedAmount = amount.div(new BN(factoring))
  return (skimmedAmount.toNumber() / 10 ** decimal).toString()

```